app/views/docs/installation.phtml

```diff
@@ -56,6 +56,14 @@
     </li>
 </ul>
 
+<p>You will be propted to configure the following during setup:</p>
+<ol class="margin-top margin-bottom-large text-size-normal">
+	<li>Your Appwrite instance's HTTP and HTTPs ports.</li>
+	<li>Your Appwrite instance's sercret key used to encrypt sensitive data.</li>
+    <li>Your Appwrite instance's hostname. Custom domains of projects created in this Appwrite instance must be a subdomain of this hostname.</li>
```

Comment:
Can we call it the "main" hostname since we call it "main domain" in the [certificate](https://appwrite.io/docs/certificates#generationCycle) docs?

Also, custom domains don't need to be a subdomain. They can be anything as long as the CNAME points to `_APP_DOMAIN_TARGET`

```suggestion
    <li>Your Appwrite instance's main hostname. Appwrite will generate a certificate using this hostname.</li>
```